src/bin/commands/bootstrap/bootstrap-stack.ts

```diff
@@ -0,0 +1,115 @@
+import { readFileSync } from "fs";
+import { StringParameter } from "@aws-cdk/aws-ssm";
+import { App } from "@aws-cdk/core";
+import AWS from "aws-sdk";
+import type { CreateStackInput } from "aws-sdk/clients/cloudformation";
+import chalk from "chalk";
+import { prompt } from "inquirer";
+import type { Question } from "inquirer";
+import type { SsmParameterPath } from "../../../constants/ssm-parameter-paths";
+import { GuStack } from "../../../constructs/core";
+import type { GuStackProps } from "../../../constructs/core";
+import type { AwsAccountReadiness } from "../../../types/cli";
+
+interface PopulatedParameter extends SsmParameterPath {
+  value: string;
+}
+
+interface BootstrapProps extends GuStackProps {
+  parameters: PopulatedParameter[];
+}
+
+class AccountBootstrap extends GuStack {
+  migratedFromCloudFormation = true;
+
+  constructor(scope: App, id: string, props: BootstrapProps) {
+    super(scope, id, props);
+
+    this.templateOptions.description = `CDK Bootstrap template used to manage required SSM parameters and other resources.`;
+
+    props.parameters.forEach((p) => {
+      new StringParameter(this, p.path, {
+        description: p.description,
+        parameterName: p.path,
+        stringValue: p.value,
+      });
+    });
+  }
+}
+
+export const accountBootstrapCfn = async (parameters: SsmParameterPath[]): Promise<string> => {
+  // Inquirer interprets '.' in question names as a special character, so let's
+  // replace it.
+  const safeName = (name: string): string => name.replaceAll(".", "-");
+
+  const stackQ: Question = {
+    type: "string",
+    name: "stack",
+    message:
+      "Enter a Guardian stack, used to tag resources (e.g. 'frontend' - typically the same as your Janus profile)",
+  };
```

Comment:
This isn't always true (e.g. profile=deployTools, stack=deploy) and might actually be thought of as redundant as we're already required to pass a `--profile` flag to this command.

Reply:
Yeah, it would be nice if it was always the same as profile but unfortunately not!